src/services/lnd/onchain-service.ts

```diff
@@ -136,13 +140,39 @@ export const OnChainService = (
     txHash,
     scanDepth,
   }: LookupOnChainFeeArgs): Promise<Satoshis | OnChainServiceError> => {
-    const onChainTxs = await listOutgoingTransactions(scanDepth)
+    const onChainTxs = await listOutgoingTransactions({ scanDepth })
```

Comment:
Can we call `lookupOnChainPayment` here instead and simply pass forward `args` to be more direct?

Reply:
Updated https://github.com/GaloyMoney/galoy/pull/1916/commits/ccf62bb393a1e6e0541e9fdb7860869cd3063813